player/public/js/baseLauncher.js

```diff
@@ -217,7 +217,7 @@ org.ekstep.contentrenderer.baseLauncher = Class.extend({
 				type: "error",
 				custom: { timeOut: 200000 },
 				errorInfo: {
-					errorStack: errorObj.stack,
+					errorStack: errorObj,
```

Comment:
Have you verified this. Always are we getting as error object?

Reply:
Yes, I have verified this. This errorObj is input to the taster plugin, where we need an error object, not the error stack.